bcrypt-addon/src/main/java/eu/europeana/keycloak/password/BCryptPasswordHashProvider.java

```diff
@@ -19,16 +20,26 @@
  */
 public class BCryptPasswordHashProvider implements PasswordHashProvider {
 
+    private static final int     HARDCODEDNROFITERATIONS = 13;
```

Comment:
It customary to use underscores for static finals for better readability, e.g.
HARD_CODED_NR_OF_ITERATIONS